harvard-lts/CURIOSity
# Adding new skip link for item detail pages.

**Adding new skip link for item detail pages.**
* * *

**JIRA Ticket**: [LTSMAINT-758](https://jira.huit.harvard.edu/browse/LTSMAINT-758)

# What does this Pull Request do?
This adds a new skip link to the top of item detail pages only. The label of the link is "" and if you click it, then it drops you down to the section of the page with the item metadata. The skip link does not appear on homepage, search results or feature pages etc.

# How should this be tested?

A description of what steps someone could take to:
* Rebuild the application with the `LTSMAINT-758` branch
* Go to an item page, for example: https://localhost:21407/demo/catalog/47-990014631980203941
* Press the tab key three times. You should see 3 skip links appear: "Skip to Search", "Skip to Main Content" and "Skip to Item description"
* If you click on "Skip to item description" it should send you down to the the portion of the page with the item metadata (below the viewer).
* The "Skip to item description" skip link should not appear on any other pages of the site besides item detail pages.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz @dl-maura @mferrarini
